examples/examples-cpu/nyc-taxi/hyperparameter-scikit.ipynb

```diff
@@ -54,18 +54,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n",
     "import numpy as np\n",
-    "import datetime\n",
     "import pandas as pd\n",
-    "import s3fs\n",
     "import warnings\n",
     "warnings.simplefilter(\"ignore\")\n",
     "\n",
-    "s3 = s3fs.S3FileSystem(anon=True)\n",
-    "\n",
     "taxi = pd.read_csv(\n",
-    "    s3.open('s3://nyc-tlc/trip data/yellow_tripdata_2019-01.csv', mode='rb'),\n",
+    "    'https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv',\n",
```

Comment:
since these `-scikit` examples are just reading a single CSV file into a pandas dataframe, I don't think we're getting any benefit from using `s3fs`

Reply:
Agree.

Reply:
s3fs is required for anonymous connections. If a user doesn't have AWS credentials setup, regular `pd.read_csv()` will fail

Reply:
that's not true.

Here's an example in an environment with no AWS creds and no `s3fs`:

```shell
docker run -it continuumio/miniconda3 /bin/bash

pip install pandas
```

```python
import pandas as pd
df = pd.read_csv(
    "https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv"
)
df.head()
```

![image](https://user-images.githubusercontent.com/7608904/99727803-1286c480-2a7e-11eb-8b27-fe63dfbc6952.png)

```python
import s3fs
```

![image](https://user-images.githubusercontent.com/7608904/99727868-28948500-2a7e-11eb-9938-83288428301e.png)

Reply:
notice that in this PR, I also changed the protocol from `s3://` to `https://`. You probably ran into credentials issues before because of the combination of no creds AND using `s3://`. But every file in S3 is available over HTTPS too, and public ones don't require any authentication :grinning:

Reply:
🤯🤯

Reply:
TIL. I'll make sure to modify other code we have around that uses `s3://`